ember-cli-build.js

```diff
@@ -17,9 +21,35 @@ module.exports = function() {
   ].map(packageDist);
 
   vendorTrees.push(buildVendorPackage('simple-html-tokenizer'));
-  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), { 
+  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), {
     include: ['dist/handlebars.amd.js']
   }));
 
-  return build({ vendorTrees });
-}
\ No newline at end of file
+  let tsAndJs = funnel(__dirname, {
+    include: ['src/**/*.ts', 'src/**/*.js']
+  });
+
+  let templates = funnel(__dirname, {
+    include: ['src/**/*.hbs']
+  });
+
+  let compiledTemplates = new GlimmerTemplatePrecompiler(templates, {
+    rootName: '-application'
```

Comment:
This smells a bit off to me. Are we bringing in the `GlimmerTemplatePrecompiler` here just to process src/templates/main.hbs? Could we instead precompile this template along with the app's templates in the application-pipeline? I haven't pulled this down yet and might be missing something.